bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -280,6 +283,26 @@ void listenAndAnswer(@Nullable STTService stt, @Nullable TTSService tts, @Nullab
      */
     Collection<KSService> getKSs();
 
+    /**
+     * Retrieves a HumanLanguageInterpreter collection.
+     * If no services are available returns null.
+     * 
+     * @param ids Comma separated list of HLI service ids to use
+     * @return a List<HumanLanguageInterpreter> or null, if none of the services is available
+     */
+    @Nullable
+    List<HumanLanguageInterpreter> getHLIsByIds(String ids);
```

Comment:
As I said earlier, why do you return `null` and not an empty list? Checking `list.isEmpty()` or `list == null` is the same, but avoids unnecessary null.